src/main.rs

```diff
@@ -150,9 +146,7 @@ fn main() {
         }
     });
 
-    rocket::ignite()
-        .mount("/", StaticFiles::from("/home/pi/web").rank(30))
```

Comment:
not sure if you actually had files here, but that import doesn't work with new rocket